service/common/build.gradle.kts

```diff
@@ -90,6 +90,8 @@ dependencies {
   implementation("com.azure:azure-storage-blob")
   implementation("com.azure:azure-storage-file-datalake")
 
+  compileOnly(libs.microprofile.fault.tolerance.api)
```

Comment:
Should this be an `implementation` dep because the exception mapper references these classes?